modules/ROOT/pages/conf-examples/office-integration.adoc

```diff
@@ -0,0 +1,124 @@
+
+* If you want to use the Traefik dashboard, set `TRAEFIK_DASHBOARD` to `true` (default is `false` and therefore not active). If you activate it, you must set a domain for the Traefik dashboard in `TRAEFIK_DOMAIN=`, for example `TRAEFIK_DOMAIN=traefik.owncloud.test`.
+
+* The Traefik dashboard is secured by Basic Auth. Default credentials are the user `admin` with the password `admin`. To set your own credentials, generate a htpasswd (e.g. by using an https://htpasswdgenerator.de/[online tool] or a CLI tool).
+
+* Traefik will issue certificates with LetsEncrypt. Therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
+
+* By default Infinite Scale will be started with the latest version. If you want to run a specific version of Infinite Scale, set the version to `OCIS_DOCKER_TAG=`. Available versions can be found on https://hub.docker.com/r/owncloud/ocis/tags?page=1&ordering=last_updated[Docker Hub].
+
+* Set your domain for the Infinite Scale web frontend in `OCIS_DOMAIN=`, for example `OCIS_DOMAIN=ocis.owncloud.test`.
+
+* Set the initial admin user password in `ADMIN_PASSWORD=`. It defaults to `admin`.
+
+* By default, the CS3Org WOPI server will be started with version `v9.3.0`. If you want to start a different or more recent version, you can set the `WOPISERVER_DOCKER_TAG=`. Available versions can be found on {wopiserver_url}[Docker Hub].
+
+* Set your domain for the CS3Org WOPI server in `WOPISERVER_DOMAIN=`, where all office suites can download the files via the WOPI protocol.
+
+* You also must override the default WOPI JWT secret for a secure setup. Do this done by setting `WOPI_JWT_SECRET` to a long and random string.
+
+* For OnlyOffice, wopi is not enabled by default. It has to be set explicit via `WOPI_ENABLED: "true"`.
+
+* To set up Collabora, you need to specify the domain of Collabora in `COLLABORA_DOMAIN=`.
+
+* If you want to use the Collabora admin panel, you need to set the username and password for the administrator in `COLLABORA_ADMIN_USER=` and `COLLABORA_ADMIN_PASSWORD=`.
+
+* Next, enter the OnlyOffice domain in `ONLYOFFICE_DOMAIN=`.
+--
+
+. Though no changes are needed, adapt the xref:conf-examples/ext-files/list_file_2.adoc[UI representation,window=_blank] of the office applications for the use in ownCloud Web if required. As an example, the settings do currently not have any icons defined that show up in the webUI with the names of available office applications to open. This can be changed manually.
+
+== Start the Docker Stack
+
+[start=4]
+. When the changed files have been saved, start the Docker stack with:
++
+[source,bash]
+----
+docker-compose up -d
+----
+
+== Access Infinite Scale
+
+[start=5]
+. Enter your Infinite Scale domain in a browser and you're ready to log in and open office documents in the web client. It might take a few minutes for all services to become operational because the container need to download first. Just keep reloading the pages from time to time.
```

Comment:
```suggestion
. Enter your Infinite Scale domain in a browser and you're ready to log in and open office documents in the web client. It might take a few minutes for all services to become operational because the containers need to download first. Just keep reloading the pages from time to time.
```